Mathlib/Topology/Algebra/Module/Equiv.lean

```diff
@@ -1028,15 +1028,17 @@ variable [AddCommGroup M] [Module R M]
 variable [AddCommGroup M₂] [Module R M₂]
 
 @[simp]
-theorem ring_inverse_equiv (e : M ≃L[R] M) : Ring.inverse ↑e = inverse (e : M →L[R] M) := by
+theorem ringInverse_equiv (e : M ≃L[R] M) : Ring.inverse ↑e = inverse (e : M →L[R] M) := by
   suffices Ring.inverse ((ContinuousLinearEquiv.unitsEquiv _ _).symm e : M →L[R] M) = inverse ↑e by
     convert this
   simp
   rfl
 
+@[deprecated (since := "2025-04-22")] alias ring_inverse_equiv := ringInverse_equiv
+
 /-- The function `ContinuousLinearEquiv.inverse` can be written in terms of `Ring.inverse` for the
 ring of self-maps of the domain. -/
-theorem to_ring_inverse (e : M ≃L[R] M₂) (f : M →L[R] M₂) :
+theorem to_ringInverse (e : M ≃L[R] M₂) (f : M →L[R] M₂) :
```

Comment:
When reading that name, I stumbled over the spelling... while `toRingInverse` is definitely wrong, the current name also feels strange. This is pre-existing, so no need to change.

Reply:
No, you're right, I had the same feeling. Fixed